travis-run-tests.sh

```diff
@@ -1,22 +1,24 @@
 #! /usr/bin/env bash
 
-set -x
-
 current_version=`python setup.py --version`
-if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" != false ]]
+
+if ! git diff-index --quiet master openfisca*/**
```

Comment:
I think this line is sufficient:

```
git diff-index master openfisca_core
```

because:

```
~/D/o/openfisca-core (master) $ echo a >> openfisca_core/entities.py 
~/D/o/openfisca-core (master) $ echo a >> openfisca_core/scripts/json_with_test_case_to_input_variables.py 
~/D/o/openfisca-core (master) $ git diff-index master openfisca_core
:100644 100644 9f3ea7f0ae76e7a8d728c0ddc80b383ca10ca948 0000000000000000000000000000000000000000 M	openfisca_core/entities.py
:100755 100755 b34f6d077c0b2dff7f4e551ae0a0a0a6cfc091d3 0000000000000000000000000000000000000000 M	openfisca_core/scripts/json_with_test_case_to_input_variables.py

```